tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -338,6 +339,14 @@ public bool ValidateRepoInformation()
                 return false;
             }
 
+            var sshMatch = Regex.Match(Url, "^.*@.*:.*\\/.*[.].*");

```

Comment:
IIRC the `.git` suffix is optional. Can be simplified to:
```suggestion
            var sshMatch = Regex.Match(Url, "^.*@.*:.*/.*");
```

Reply:
Still need to escape the delimiter.  Suggestion added.